libs/backend/src/cast_vote_records/import.ts

```diff
@@ -167,10 +167,8 @@ async function* castVoteRecordGenerator(
         !castVoteRecord.BallotImage[1] ||
         !castVoteRecord.BallotImage[0].Hash?.Value ||
         !castVoteRecord.BallotImage[1].Hash?.Value ||
-        // These next two conditions are hard to trigger without first triggering the invalid
-        // write-in case above.
-        /* istanbul ignore next */ !castVoteRecord.BallotImage[0].Location ||
-        /* istanbul ignore next */ !castVoteRecord.BallotImage[1].Location ||
+        !castVoteRecord.BallotImage[0].Location ||
+        !castVoteRecord.BallotImage[1].Location ||
         !castVoteRecord.BallotImage[0].Location.startsWith('file:') ||
         !castVoteRecord.BallotImage[1].Location.startsWith('file:')
```

Comment:
Some interesting behavior here that I haven't fully grok'd, but I actually only needed the ignores when I was using optional chaining here, before this PR, e.g.

<img width="610" alt="coverage" src="https://github.com/votingworks/vxsuite/assets/12616928/541d35ff-c3d6-4a11-8b1b-f46ab7e65cc7">

When I get rid of the optional chaining, I hit 100% coverage, no ignores or updates to tests needed.